lib/libvarnishapi/vsm.c

```diff
@@ -484,8 +484,25 @@ vsm_findcluster(const struct vsm_set *vs, const char *cnam)
 	return (NULL);
 }
 
+static unsigned
+vsm_running(struct vsm_set *vs, pid_t pid)
+{
+
+	AN(vs);
+	assert(pid > 1);
```

Comment:
The assertion on `pid` triggered in CI, so it should probably be relaxed and turned into a "not running" outcome. I will try to understand why.